runtime/doc/treesitter.txt

```diff
@@ -390,14 +390,17 @@ The following directives are built in:
             (#gsub! @_node ".*%.(.*)" "%1")
 <
     `trim!`                                          *treesitter-directive-trim!*
-        Trim blank lines from the end of the node. This will set a new
-        `metadata[capture_id].range`.
+        Trim whitespace from the node. This will set a new
+        `metadata[capture_id].range`. Trims blank lines from the end of the node
+        by default (for backwards compatibility). Trims all whitespace from both
+        sides of the node if the optional parameter "1" is given.
```

Comment:
this doc needs to be updated now, right?
```suggestion
        sides of the node if parameters are given.
```

Reply:
It does; sorry maybe I should mark as draft (even though this is 90% done?)- still working through a test failure which is causing me headache